.github/workflows/test_iree.yml

```
@@ -101,17 +101,23 @@ jobs:
           - name: cpu_llvm_sync
```

I noticed this after merging, but I should have named this `cpu_llvm_task`. Could do that in a new PR or here.
```suggestion
          - name: cpu_llvm_task
```

---

made change here